lib/connection/bind_uploader.js

```diff
@@ -69,27 +75,14 @@ function BindUploader(options, services, connectionConfig, requestId)
 			}
 			strbuffer += '\n';
 
-			if (curBytes < MAX_BUFFER_SIZE)
-			{
-				var size = Buffer.byteLength(strbuffer, 'utf8');
-				curBytes += size;
-			}
-			else
+			if ((strbuffer.length >= MAX_BUFFER_SIZE) || (i == bindings.length -1))
 			{
-				var fileName = (++fileCount).toString();
-				Logger.getInstance().debug('fileName='+fileName);
+				var fileName = tmpDir + "\\" + (++fileCount).toString();
```

Comment:
Do we need OS specific path separator? why using "\\"?

Reply:
tmpDir is the folder path, need \ for the file path.